template/package.json

```
@@ -193,7 +193,7 @@
         "changelogFile": "CHANGELOG.md"
       }
     ],
-    "@semantic-release/npm",
+    "@semantic-release/<%= npmClient %>",
```

this should still be npm :)
```suggestion
    "@semantic-release/npm",
```